.pre-commit-config.yaml

```diff
@@ -1,16 +1,16 @@
 repos:
     - repo: https://github.com/psf/black
-      rev: 24.8.0
+      rev: 24.10.0
```

Comment:
This should just be synced in pyproject.toml files:
currently `black = "~24.8.0"`
and in lint workflow